Quick_Deploy/vLLM/model_repository/vllm/1/model.py

```diff
@@ -190,6 +190,9 @@ async def generate(self, request):
                     response_sender.send(self.create_response(output))
                 else:
                     last_output = output
+                if response_sender.is_cancelled():
```

Comment:
Can we move this section just at the beginning of the loop? No need to create and send a response for a cancelled request. We should avoid as much extra work as possible.

Reply:
Yes: [Move is_cancelled() check to top](https://github.com/triton-inference-server/tutorials/pull/56/commits/7b7259912ed6035f1840f1cde76943a7501d405b)